### ISSUE TITLE
Make it possible to disable editors following the highlighted entry in the problems list

### ISSUE THREAD

**Issue by webczat** [microsoft/vscode#110215]
Currently, when I have problems shown in the problems view, navigating through the error list with just arrows immediately moves the cursor to their location, and opens new editors if needed.
However this poses some problems related to accessibility, so I would like to suggest a feature allowing this behavior to be disabled.
In case this setting is disabled, the cursor should move only when pressing enter on a specific problem.
The motivation for this request is the fact this behavior is bad for screenreader usage.
Normally (possibly except mac) blind people navigate using arrows, so in order to see any problem list item we have to actually select it first with arrows.
And in that case I would not expect being moved to location of each problem I select when I only wanted to review the list.

**Comment by isidorn** [comment]
This is a fair feature request and the one I discussed with @sandy081 thus forwarding to him

**Comment by jvesouza** [comment]
This is a great suggestion.

**Comment by MarcoZehe** [comment]
FWIW, when on Mac, and the default of "VoiceOver cursor moves focus" is on, moving to a problem with the VoiceOver commands also makes the focus jump and the context to get lost, and usually, this also means that VoiceOver does not read the problem text because it gets interrupted by itself announcing the new focus position in the editor. So, I definitely support this feature request!

**Comment by isidorn** [comment]
Agree would be great to have this.
